Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN yum -y install centos-release-scl-rh && \
                                            samba-common                 \
                                            samba-common-tools           \
                                            && \
+    yum -y install --setopt=tsflags=nodocs mod_ssl                      \
```

Comment:
Does this apply to a particular auth mechanism? (Should it be in one of the above commands instead).

Reply:
I don't think so from my understanding. This more opens up the possibility to have SSL vhosts when laying down an `application.conf` file for example. This is the one thing missing to allow Foreman in containers to use this directly instead of carrying around a "patched" version.

Reply:
we had mod_ssl earlier, but dropped for preferences of dealing with SSL by the openshift route definition and run the httpd pod and backends with non-ssl.  Won't this approach work with Foreman ?

If not, I don't mind bringing this back in here.  Maybe move it to before the auth specific rpm's (right before SSSD stuff).

Reply:
Foreman has some high SSL demands. Using SSL termination on Openshift routes does not work for it because there is information within the certificates that needs to make to the application itself.